developers/academy/js/starter_multimodal_data/103_mm_searches/30_filters.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Filters
+description: Filters for Multimodal Searches
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import TSCode from '!!raw-loader!../_snippets/103_searches.mts';
+
+Filters can be used to precisely refine search results. You can filter by properties as well as metadata, and you can combine multiple filters with `and` or `or` conditions to further narrow down the results.
+
+### <i class="fa-solid fa-code"></i> Code
+
+This example finds entries in "Movie" based on their similarity to the query "dystopian future", only from those released after 2010. It prints out the title and release year of the top 5 matches.
+
+<FilteredTextBlock
+  text={TSCode}
+  startMarker="// FilteredSemanticSearch"
+  endMarker="// END FilteredSemanticSearch"
+  language="ts"
+/>
+
+## <i class="fa-solid fa-chalkboard"></i> Explain the code
+
+This query is identical to [that shown earlier](./10_multimodal.mdx) for search, but with the addition of a filter. The `filters` parameter here takes an instance of the `Filter` class to set the filter conditions. The current query filters the results to only include those with a release year after 2010.
```

Comment:
Similarly to above - "Filter" class seems specific to Python only

Reply:
fixed!!